src/exercise/dist/clients/item_client.js

```diff
@@ -0,0 +1,47 @@
+// Create an ItemClient class here. This is what makes requests to your express server (your own custom API!)
+
+// const axios = require('axios').default;
+
+class ItemClient {
+    constructor() {
+        this.API_BASE = 'http://localhost:8080/task';
+    }
+
+    async getTasks(){
+        const tasks = await axios.get(this.API_BASE);
+        return tasks;
+    }
+
+    async addTask(task, date){
+        try{
+            await axios({
+                method: 'post',
+                url: this.API_BASE,
+                data: {
+                    "task": task,
+                    "date": date
+                },
+                headers: {
+                    'Content-Type': 'application/json'
+                }
+            });
+        } catch(error){
```

Comment:
not bad, however, axios has a [built-in error handling](https://axios-http.com/docs/handling_errors) way of doing things